packages/chart/src/index.ts

```diff
@@ -1,12 +1,12 @@
-export { default as Chart } from './Chart';
+export { default as Chart } from './LazyChart';
 export { default as ChartModelFactory } from './ChartModelFactory';
 export { default as ChartModel } from './ChartModel';
 export { default as ChartUtils } from './ChartUtils';
 export * from './ChartUtils';
 export * from './DownsamplingError';
 export { default as FigureChartModel } from './FigureChartModel';
 export { default as MockChartModel } from './MockChartModel';
-export { default as Plot } from './plotly/Plot';
+export { default as Plot } from './plotly/LazyPlot';
```

Comment:
Are there any breaking change implications of changing default imports to lazy ones? The only one I can think of is does this now require a Suspense boundary in places that wouldn't have required one before?

Reply:
Oh nvm, looks like you baked the Suspense boundary into the exported default.

Reply:
Yep it should be non-breaking since `Suspense` and `lazy` has existed since React 16 and we specify/use 17